src/lib/viewers/BaseViewer.js

```diff
@@ -1084,6 +1086,31 @@ class BaseViewer extends EventEmitter {
         this.emit('annotatorevent', data);
     }
 
+    /**
+     * Creates Intl object used by annotations
+     *
+     * @private
+     * @return {Object}
+     */
+    createAnnotatorIntl() {
```

Comment:
Can we create a small i18n helper module to hide these details, similar to our other apps?